app/models/edition/custom_lead_image.rb

```diff
@@ -23,8 +25,12 @@ def non_lead_images
 
 private
 
-  def oldest_image
-    images.order(:created_at, :id).first
+  def non_svg_images
+    images.reject { |image| image.content_type == SVG_CONTENT_TYPE }
```

Comment:
Would it make any sense to have this be a scope on the Image model and then use the scope in this class?

Reply:
Yea what makes this tricky is that the content type lives of the file object generated via carrierwave. We could add a scope like 

```
scope :svg, -> { joins(:image_data).where("carrierwave_image LIKE '%.svg'") }
```

but it feels a bit gross. Wdyt?

Reply:
Ah ok. Are we in any danger of N+1 queries here, in that case, or is the `image_data` table always eager loaded?

Reply:
I might just be a bit clueless on carrierwave 😂 but i think it's more that we don't have the content type persisted to the database. Carrierwave grabs the asset from asset manager and provides us access to the content_type as part of the API.

Reply:
I'm having a proper dig into this, bear with me...